cmd/fetchd/cmd/stakereconciliationmigrate.go

```diff
@@ -1,315 +0,0 @@
-package cmd
-
-import (
-	"encoding/csv"
-	"encoding/json"
-	"fmt"
-	"io"
-	"io/ioutil"
-	"os"
-	"strings"
-
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/codec"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/errors"
-	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
-	"github.com/spf13/cobra"
-	tmjson "github.com/tendermint/tendermint/libs/json"
-
-	"github.com/fetchai/fetchd/app"
-)
-
-type IneligibleRegistration struct {
-	NativeAddr string
-	Reason     error
-}
-
-func (i IneligibleRegistration) Print() {
-	fmt.Fprintf(os.Stderr, "%q ineligible for reason: %s\n", i.NativeAddr, i.Reason.Error())
-}
-
-type Registration struct {
-	EthAddress    string `json:"eth_address"`
-	NativeAddress string `json:"native_address"`
-}
-
-type Registrations []Registration
-
-type AddrString = string
-type AddrMap map[AddrString]Registration
-
-func (r Registrations) EthAddrMap() AddrMap {
-	regMap := make(AddrMap)
-	for _, reg := range r {
-		regMap[normalizeEthAddr(reg.EthAddress)] = reg
-	}
-	return regMap
-}
-
-var (
-	errNonZeroSeqNum   = fmt.Errorf("%s", "sequence number must be 0")
-	errBalanceNotMatch = fmt.Errorf("%s", "old account balance must match staked export amount")
-	stakesCSVPath,
-	registrationsPath,
-	coinDenom string
-	debugFlag    bool
-	skipValidate bool
-)
-
-func AddStakeReconciliationMigrateCmd() *cobra.Command {
```

Comment:
We should keep this command and its dependencies. While we won't do it this upgrade. The plan is certainly to do an update for the next upgrade so this will be reused